test/parser.test.coffee

```diff
@@ -156,11 +156,30 @@ module.exports =
     equ r.sample.$$[10].$$[2]._, '3'
     equ r.sample.$$[10].$$[3]['#name'], 'one'
     equ r.sample.$$[10].$$[3]._, '4'
-    equ r.sample.$$[10].$$[4]['#name'], 'two'
+    equ r.sample.$$[10].$$[4]['#name'], 'two' 
     equ r.sample.$$[10].$$[4]._, '5'
     equ r.sample.$$[10].$$[5]['#name'], 'three'
     equ r.sample.$$[10].$$[5]._, '6')
 
+  'test non-async parsing': (test) ->
+    x2js = new xml2js.Parser()
+    data = fs.readFileSync fileName, 'utf8'
+    xmldoc = x2js.parseStringSync data
```

Comment:
By `yet-to-be-added-shorthand` do you mean importing `parseString` directly from the `require`?